elixir/apps/domain/priv/repo/migrations/20230825174719_update_policy_indexes.exs

```diff
@@ -0,0 +1,25 @@
+defmodule Domain.Repo.Migrations.UpdatePolicyIndexes do
+  use Ecto.Migration
+
+  def up do
+    drop(index(:policies, [:account_id, :name]))
+    drop(index(:policies, [:account_id, :resource_id, :actor_group_id]))
+
+    create(index(:policies, [:account_id, :name], unique: true, where: "deleted_at IS NULL"))
+
+    create(
+      index(:policies, [:account_id, :resource_id, :actor_group_id],
+        unique: true,
+        where: "deleted_at IS NULL"
+      )
+    )
+  end
+
+  def down do
+    drop(index(:policies, [:account_id, :name]))
+    drop(index(:policies, [:account_id, :resource_id, :actor_group_id]))
+
+    create(index(:policies, [:account_id, :name], unique: true))
+    create(index(:policies, [:account_id, :resource_id, :actor_group_id], unique: true))
+  end
```

Comment:
ATM we don't support rollbacks:
```suggestion
  def change do
    drop(index(:policies, [:account_id, :name]))
    drop(index(:policies, [:account_id, :resource_id, :actor_group_id]))

    create(index(:policies, [:account_id, :name], unique: true, where: "deleted_at IS NULL"))

    create(
      index(:policies, [:account_id, :resource_id, :actor_group_id],
        unique: true,
        where: "deleted_at IS NULL"
      )
    )
  end
```

Reply:
Ah, ok, I didn't realize we weren't supporting rollbacks.  I actually started with a `def change do` 😆

Reply:
This has been updated